Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && \
+    apt-get install -y \
+      python \
+      python-pip \
+      libcapstone-dev \
+    && \
+    rm -fr /var/lib/apt/lists/*
+
+RUN pip install 'capstone>=3.0,<4.0'
+
+COPY . /src/github.com/xoreaxeaxeax/sandsifter
+WORKDIR /src/github.com/xoreaxeaxeax/sandsifter
+
+RUN make
+
+ENTRYPOINT ["/usr/bin/python", "/src/github.com/xoreaxeaxeax/sandsifter/sifter.py"]
```

Comment:
I think it would be a good idea to have an entrypoint that is a shell-script using this as one of many optional actions, provisioning for arbitrary container command execution.